packages/docs/src/pages/api-styled.mdx

```diff
@@ -17,6 +17,8 @@ import {
 
 Create a component that styles a JSX element which comes with built-in behavior such as `ref` and `as` prop support.
 
+<StyledObj />
+<br />
 <StyledString />
```

Comment:
What if
- we remove `<StyledString />` example
- add something along the lines 'call expression syntax is preferred ( [no-styled-tagged-template-expression](https://github.com/atlassian-labs/compiled/tree/master/packages/eslint-plugin/src/rules/no-styled-tagged-template-expression))'

Reply:
Sounds good 👍 Should I delete the code for `StyledString`?

Reply:
yeahhh, let's remove line 22, but keep the code for `StyledString`